<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KY Editor</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="tabChangesFocus">
         <bool>false</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew_File_Ctrl_N"/>
    <addaction name="actionOpen_File_Ctrl_O"/>
    <addaction name="actionSave_Ctrl_s"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionClose_Window"/>
    <addaction name="actionClose_File"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuColors">
     <property name="title">
      <string>Colors</string>
     </property>
     <addaction name="actionBlack"/>
     <addaction name="actionWhite"/>
    </widget>
    <addaction name="actionSyntex"/>
    <addaction name="menuColors"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build(&amp;B)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuBuild"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_File_Ctrl_N">
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File_Ctrl_O">
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionSave_Ctrl_s">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionClose_Window">
   <property name="text">
    <string>Close Window</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionSyntex">
   <property name="text">
    <string>Syntex</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
